config.sample.js

```diff
@@ -2,3 +2,4 @@ export const CLIENT_ID = ''
 export const CLIENT_SECRET = ''
 export const SESSION_SECRET = ''
 export const CURRENT_ELECTION_INFORMATION_TIMEOUT = 60
+export const ADMIN_USERGROUP_ID = 1
```

Comment:
Just in case someone wants to set this to something else, I put it in the config. I also added a script to create this in the DB, so I hope people will do it right after running all the migrations.